wowup-electron/src/app/components/addon-update-button/addon-update-button.component.ts

```diff
@@ -17,7 +17,7 @@ import { getEnumName } from "../../utils/enum.utils";
   styleUrls: ["./addon-update-button.component.scss"],
 })
 export class AddonUpdateButtonComponent implements OnInit, OnDestroy {
-  @Input() public listItem: AddonViewModel;
+  @Input() public listItem!: AddonViewModel;
```

Comment:
the ! says it will have a value when we go to access it. so the type checking does not complain about this not having an initial value. 

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator